Source/Core/Core/IOS/USB/Bluetooth/BTReal.h

```diff
@@ -58,7 +58,9 @@ class BluetoothRealDevice final : public BluetoothBaseDevice
   void HandleCtrlTransfer(libusb_transfer* finished_transfer);
   void HandleBulkOrIntrTransfer(libusb_transfer* finished_transfer);
 
+  static bool IsConfiguredBluetoothDevice(libusb_device_descriptor device_descriptor);
```

Comment:
`libusb_device_descriptor` is a decently-sized structure. I recommend you pass it as a const-reference or just use the existing vid/pid version with `(device_descriptor.idVendor, device_descriptor.idProduct)` at the new call-site.